eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -13,6 +18,11 @@
 {-# LANGUAGE UndecidableInstances #-}
 {-# LANGUAGE UndecidableSuperClasses #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
+#if __GLASGOW_HASKELL__ < 963
+-- `ToExpr (PParamsHKD Identity era)` is reported as redundant otherwise
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+#else
+#endif
```

Comment:
CPP should be the last resort in solving a problem
```suggestion
```

Add it to `ShelleyEraImp` type class as superclass constraint that will solve your problem.